packages/beacon-node/src/chain/archiver/archiveStates.ts

```diff
@@ -43,22 +47,23 @@ export class StatesArchiver {
   async maybeArchiveState(finalized: CheckpointWithHex): Promise<void> {
     const lastStoredSlot = await this.db.stateArchive.lastKey();
     const lastStoredEpoch = computeEpochAtSlot(lastStoredSlot ?? 0);
+    const archiveStateEpochFrequency = this.opts.archiveStateEpochFrequency;
```

Comment:
could be destructured, depends on what you prefer :)

```suggestion
    const {archiveStateEpochFrequency} = this.opts;
```